tests/suites/test_suite_x509parse.function

```diff
@@ -434,8 +471,10 @@ void x509_verify_ca_cb_failure( char *crt_file, char *ca_file, char *name,
 
     if( strcmp( name, "NULL" ) == 0 )
         name = NULL;
-    
-    ret = mbedtls_x509_crt_verify_with_cb( &crt, ca_callback_fail, &ca, &compat_profile, name, &flags, verify_all, NULL );
+
+    ret = mbedtls_x509_crt_verify_with_cb( &crt, ca_callback_fail, &ca,
+                                           &compat_profile, name, &flags,
+                                           verify_all, NULL );
```

Comment:
Very minor: I don't think there's a point in passing `verify_all` rather than `NULL` here.

Reply:
Changed to NULL